YIF.Core.Data/YIF.Core.Data.csproj

```diff
@@ -4,6 +4,11 @@
     <TargetFramework>netcoreapp3.1</TargetFramework>
   </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Remove="Migrations\20210506082404_ChangedForeignAndPrimaryKeysForLector.cs" />
+    <Compile Remove="Migrations\20210506082404_ChangedForeignAndPrimaryKeysForLector.Designer.cs" />
+  </ItemGroup>
+
```

Comment:
sorry, but why these lines were added?

Reply:
https://github.com/dotnet/efcore/issues/18381

Reply:
I suppose these lines can break creation of next migrations, investigate pls about it